content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -170,12 +220,20 @@ kubectl delete pvc task-pv-claim
 kubectl delete pv task-pv-volume
 ```
 
-Remove the file:
+If you don't already have a shell open to the Node in your cluster,
+open a new shell the same way that you did earlier.
+
+In the shell on your Node, remove the file and directory that you created:
 
```

Comment:
IMO, git rid of the unuse content

Reply:
@stewart-yu, I'm not sure what you're suggesting here. Can you clarify?

Reply:
What should change here? I'm not sure I follow - sorry!